slackistrano.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency 'capistrano', '>= 3.8.1'
+  gem.add_dependency 'capistrano', '>= 3.9.0'
```

Comment:
Is this dependency version bump necessary?

Reply:
No, it's my fault. It can be done later